tests/x11regressions/libreoffice/libreoffice_double_click_file.pm

```diff
@@ -13,6 +13,7 @@
 use base "x11regressiontest";
 use strict;
 use testapi;
+use utils;
```

Comment:
I think you have added `utils` for `sle_version_at_least`, no? In this case your lab seems a little bit outdated, as now version control functions are in `version_utils`.
And you don't have to load all the functions, you can restrict to the needed ones, like this: `use version_utils qw(is_sle sle_version_at_least);`.

Reply:
Thanks, I just fixed it, please help to review.